metricbeat/module/elasticsearch/node_stats/_meta/data.json

```diff
@@ -1,239 +1,293 @@
 {
-    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "@timestamp": "2021-05-28T12:00:06.660Z",
+    "@metadata": {
+        "beat": "metricbeat",
+        "type": "_doc",
+        "version": "8.0.0"
+    },
     "elasticsearch": {
-        "cluster": {
-            "id": "w3oo88LcQ1i-7K4f-wrEgQ",
-            "name": "docker-cluster"
-        },
         "node": {
-            "id": "EjV2AqqvQNq5ZF5cVlaPDQ",
-            "master": true,
+            "id": "f0oLpJygRp2BGpJzAbKr8w",
             "mlockall": false,
-            "name": "foo",
+            "master": true,
+            "name": "293a428a15e9",
             "stats": {
-                "fs": {
-                    "io_stats": {},
-                    "summary": {
-                        "available": {
-                            "bytes": 45897547776
+                "os": {
+                    "cgroup": {
+                        "cpuacct": {
+                            "usage": {
+                                "ns": 127648683467
+                            }
                         },
-                        "free": {
-                            "bytes": 49114263552
+                        "cpu": {
+                            "stat": {
+                                "times_throttled": {
+                                    "count": 0
+                                },
+                                "elapsed_periods": {
+                                    "count": 0
+                                }
+                            },
+                            "cfs": {
+                                "quota": {
+                                    "us": -1
+                                }
+                            }
                         },
-                        "total": {
-                            "bytes": 62725623808
+                        "memory": {
+                            "control_group": "/",
+                            "limit": {
+                                "bytes": "9223372036854771712"
+                            },
+                            "usage": {
+                                "bytes": "2062282752"
+                            }
+                        }
+                    },
+                    "cpu": {
+                        "load_avg": {
+                            "1m": 3.33
+                        }
+                    }
+                },
+                "process": {
+                    "cpu": {
+                        "pct": 7
+                    }
+                },
+                "roles": [
```

Comment:
Help for reviewers: This is the new field, all the rest of changes on the file are just unordered or have different values.